client/js/screens/Browse/styles.js

```diff
@@ -1,21 +0,0 @@
-import {StyleSheet, Dimensions} from 'react-native';
```

Comment:
was this deliberate? to get rid on the styles file?

Reply:
yes